lib/view/formulier/InvoerVelden.class.php

```diff
@@ -1181,7 +1194,7 @@ public function isPosted() {
 
 	public function getValue() {
 		if ($this->isPosted()) {
-			$this->value = filter_var($_POST[$this->name . '_new'], FILTER_SANITIZE_STRING);
+			$this->value = $_POST[$this->name . '_new'];
```

Comment:
Dit is een schoolvoorbeeld van hoe het niet moet: rauwe user input vertrouwen en verwerken. Input sanitation 101.